src/app/forms/validators/custom-validators.spec.ts

```diff
@@ -258,6 +258,24 @@ describe('numeric', () => {
   });
 });
 
+describe('defined', () => {
+  it.each([
+    [{ defined: false }, undefined],
+    [null, ''],
+    [null, null],
+    [null, 'hello world!'],
+    [null, 1234],
+  ])('should return %o for %r', (expected: null | {[index: string]: boolean}, input: any) => {
+    const definedValidator = CustomValidators.defined();
+    let form = new FormControl(input)
+    if (typeof input === 'undefined') {
```

Comment:
Just checking that `'undefined'` should be the string `'undefined'` (in quotes) here, and that we didn't mean `undefined`

Reply:
Whenever you call typeof on a variable, it returns the type as a string value